tdms/include/unit_test_utils.h

```diff
@@ -0,0 +1,41 @@
+/**
+ * @file unit_test_utils.h
+ * @brief Functions common to multiple unit tests
+ */
+#include <complex>
+
```

Comment:
P.s. we should also pull out the global TOLERANCE and define it once here in this file (probably need to find-replace).
```suggestion
#include <complex>

namespace tdms_tests {
const double TOLERANCE = 1e-16;
}
```

Reply:
Actually now I come to think of it, I'd also probably put all of these functions into the namespace.
So `tdms_tests::is_close` etc. Then it's clearly separated from any catch2 stuff.